### DISCUSSION TITLE
Alpha comparator

### DISCUSSION THREAD

**Discussion by ledlogic** [romankh3/image-comparison#249] [Ideas]
Last week, I tried an experiment using this library but it was not detecting differences at all between what seemed like obviously different images.  I did some analysis and comparison to other libraries and this one did not recognize changes in the alpha transparency.  Sometimes, as in the case of these black and white images I was comparing, the whole image was black, and the grayness was controlled solely by the alpha value.

I would suggest amending the algorithm in pixelDiff to also compute the delta in the a1 versus a2 values, where
a1 and a2 are 24 bit offsets.  After doing that then the images were percieved correctly to be different.

**Comment by romankh3** [comment]
@ledlogic, hello. If it's actual for you, we can communicate about this topic, looks like it would be intersting.
